datastore_entity_derives/src/lib.rs

```diff
@@ -234,37 +234,45 @@ pub fn datastore_managed(input: TokenStream) -> TokenStream {
                 #kind_str
             }
         }
+
+        impl datastorers::Pagable for #name {
+            fn page_size() -> Option<i32> {
+                #page_size
+            }
+        }
+
         impl #name {
             pub fn id(&self) -> &#key_field_type {
                 #self_key_field_expr
             }
 
+            fn create_query() -> datastorers::DatastorersQuery<#name> {
```

Comment:
Now I use the new query syntax in the generated getters, but I could get them to recognise the "query" method, so I had to add this one, but I guess there must be a better solution...